interfaces/builtin/opengl.go

```diff
@@ -48,6 +48,8 @@ const openglConnectedPlugAppArmor = `
 # libdrm data files
 /usr/share/libdrm/amdgpu.ids r,
 
+# nvidia container toolkit CDI library searching at top level
```

Comment:
can you expand the comment to include more of the context which is already provided in the PR description?

Reply:
@bboozzoo Thanks, I've updated - is that what you're thinking, or did you want more still ?

Reply:
this looks ok, thanks!